torchrl/objectives/dreamer.py

```diff
@@ -111,11 +139,39 @@ def __init__(
         self.free_nats = free_nats
         self.delayed_clamp = delayed_clamp
         self.global_average = global_average
+        # actor
+        self.actor_model = actor_model
+        self.value_model = value_model
+        self.model_based_env = model_based_env
+        self.imagination_horizon = imagination_horizon
+        self.discount_loss = discount_loss
+        if gamma is not None:
+            warnings.warn(_GAMMA_LMBDA_DEPREC_WARNING, category=DeprecationWarning)
+            self.gamma = gamma
+        if lmbda is not None:
+            warnings.warn(_GAMMA_LMBDA_DEPREC_WARNING, category=DeprecationWarning)
+            self.lmbda = lmbda
+        # value
+        self.value_loss_function = value_loss if value_loss is not None else "l2"
+        self.value_gamma = value_gamma
+        self.value_discount_loss = value_discount_loss
 
     def _forward_value_estimator_keys(self, **kwargs) -> None:
```

Comment:
Model and Value had pass here, and actor has this longer implementation. Do we need it here?